### PR TITLE
Add dev.alextren.Spot

### PR DESCRIPTION
**Pull request by xou816**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by xou816** [comment]
bot, build dev.alextren.Spot

**Comment by xou816** [comment]
bot, build dev.alextren.Spot

File: dev.alextren.Spot.json [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--socket=pulseaudio",
+        "--talk-name=org.freedesktop.secrets"
+    ],
+    "build-options" : {
+        "append-path" : "/usr/lib/sdk/rust-stable/bin",
+        "build-args" : [],
```
Comment (barthalion) [inline dev.alextren.Spot.json:20]:
```suggestion
```

File: dev.alextren.Spot.appdata.xml [hunk 2]
```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (barthalion) [inline dev.alextren.Spot.appdata.xml:1]: Looks like this file is not being used now in the manifest, can you remove it?

File: dev.alextren.Spot.desktop [hunk 3]
```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
```
Comment (barthalion) [inline dev.alextren.Spot.desktop:1]: Looks like this file is not being used now in the manifest, can you remove it?

File: dev.alextren.Spot.svg [hunk 4]
```diff
@@ -0,0 +1,236 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (barthalion) [inline dev.alextren.Spot.svg:1]: Looks like this file is not being used now in the manifest, can you remove it?

**Comment by xou816** [comment]
Thanks for the review @barthalion :) I thought these files were required. Just to be clear, it is fine as long they are bundled within the flatpak itself, they don't have to be at the root of the flathub repo?
I'll remove them a bit later then, thanks!

**Comment by nedrichards** [comment]
Correct, in fact it's better if they're distributed as part of the main application artifact that way it's easier for you to keep it in sync, and if anyone else wants to reuse the source code it'll be looking at its best (as those files are used by basically everything on the Linux desktop)

**Comment by xou816** [comment]
Indeed, that's good news for maintenance! Thanks for the additional info!

File: dev.alextren.Spot.svg (hunk 4 above)
Comment (xou816) [reply dev.alextren.Spot.svg]: removed it!

File: dev.alextren.Spot.desktop (hunk 3 above)
Comment (xou816) [reply dev.alextren.Spot.desktop]: done!

File: dev.alextren.Spot.json (hunk 1 above)
Comment (xou816) [reply dev.alextren.Spot.json]: fixed :)

File: dev.alextren.Spot.appdata.xml (hunk 2 above)
Comment (xou816) [reply dev.alextren.Spot.appdata.xml]: done!

**Pull request closed without merging**